test-zuul/src/main/resources/application.yml

```diff
@@ -2,12 +2,19 @@ spring:
   application:
     name: ZUUL
 server:
+  address: 127.0.0.1
   port: 8765
 eureka:
   instance:
     metadataMap:
       instanceId: ${spring.application.name}-${random.number}
+    ip-address: 127.0.0.1
+    hostname: localhost
+    prefer-ip-address: false
   client:
     enabled: true
     serviceUrl:
-      defaultZone: http://${eureka.host:localhost}:${eureka.port:1111}/eureka/
\ No newline at end of file
+      defaultZone: http://${eureka.host:localhost}:${eureka.port:1111}/eureka/
+    use-dns-for-fetching-service-urls: false
+zuul:
+  sensitive-headers: Authorization
```

Comment:
Just to be clear as I mentioned in the pull request, allowing services behind Zuul to be stateful is dangerous and can lead to unexpected consequences. Try to use Zuul only as a stateless service proxy.